tox.ini

```
@@ -15,8 +15,7 @@ usedevelop = True
 install_command = pip install -U {opts} {packages}
 deps =
   -r{toxinidir}/requirements-dev.txt
-  qiskit-terra
-  pylatexenc
+  -r docs_guidelines/requirements-extra.txt
```

```suggestion
  -r{toxinidir}docs_guidelines/requirements-extra.txt
```
This won't really make a difference, but I think it's important to make the path relative to the toxinidir similar to the line above.